source/common/common/utility.h

```diff
@@ -107,6 +107,11 @@ class DateUtil {
  */
 class StringUtil {
 public:
+  /**
+   * White space chars.
```

Comment:
s/White space/Whitespace/

not clear a comment is necessary at all though :)

Reply:
+1 just del comment

Reply:
👍